src/components/Callout/Callout.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { CalloutProps } from './Callout.types';
+import { ColorTypes, SpaceSizes } from '../../theme';
+import { getColor, getFontSize, getRadii, pxToRem } from '../../utils';
+import { Inline, Padbox } from '../layout';
+import { Text } from '../typographyLegacy';
+import { Icon } from '../Icon';
+
+const IconContainer = styled.div`
+  width: ${pxToRem(36)};
+  height: ${pxToRem(36)};
+  flex-shrink: 0;
+  background-color: ${getColor(ColorTypes.info700)};
+  border-radius: ${getRadii('circle')};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: ${getFontSize('mdPlus')};
+`;
+
+const Container = styled.div`
+  background-color: ${getColor(ColorTypes.info50)};
+  border-radius: ${getRadii('default')};
+`;
+
+const Callout: React.FC<CalloutProps> = ({ children, icon }) => (
```

Comment:
We should have some default icons so we don't need to define new icons with every usage. Please follow the discussion in Figma.

Reply:
@julesbarr Let's add this icon in the DS icons
https://github.com/FortAwesome/Font-Awesome/blob/5.x/js/regular.js#L230
https://fontawesome.com/v5/icons/lightbulb?s=regular

and  use it as the default icon